queries.sql

```diff
@@ -0,0 +1,11 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+/*  DAY 1 */
+SELECT * FROM animals WHERE name iLIKE '%mon';
```

Comment:
Kindly check this query iLIKE, it seems there is an error here... 🥺

Reply:
> Kindly check this query iLIKE, it seems there is an error here... 🥺

- [ ] I would implore you to check this query again as suggested by the previous reviewer, do you mean LIKE?